hugo/config.toml

```diff
@@ -6,7 +6,7 @@ publishDir = "../public"
 disableKinds = ["taxonomy","taxonomyTerm"]
 
 [permalinks]
-  blog = "/blog/:title/"
```

Comment:
We should have a redirect in case someone has hardcoded the old name somewhere

Reply:
Good idea. 
:thinking: It looks like we could just use:
<code>
blog = "/news/:title/"
</code>

Correct?

Reply:
I have never done that so I don't know ^^
But it should be easily testable. Just try it out, run the hugo webserver locally and check if the redirect is working as intended :point_up:

Reply:
I don't see instructions for that.
So I will surely not look through all of this to find out how to build it exactly.